### ISSUE TITLE
How to start from scratch?

### ISSUE THREAD

**Issue by iahunq** [JulNadeauCA/BSDBuild#7]
What I found is if I run `make configure`, it will complain that `Makefile.config` is not found. This means I have to run the `configure` script to generate `Makefile.config`. But I have no `configure` script, this is the reason why I attempted to run `make configure` in the first place. Are you joking with me?

I ended up have to run these commands manually to have a `configure` script (they are extracted from `Makefile`):

```
cat configure.in | mkconfigure > configure
chmod 755 configure
```

Then I have to run this `configure` script to have `Makefile.config` so I will be able to run `make configure`. How ridiculous it is!

**Comment by iahunq** [comment]
I tried to hack. I used `touch Makefile.config` to have a `Makefile.config` file. The goal is for me able to run `make configure`. This file will be overwritten by `make configure` anyway, isn't it? Then it said I needs to have `.depend` file too. This file will also be overwritten by `make configure`, isn't it? OK. I used `touch .depend` to create that file. Now I'm able to run `make configure`. But to my surprise, it seems `.depend` is not get overwritten by `make configure` as its size is still 0. I didn't check if `Makefile.config` is updated by `make configure` or not but I think it's enough for me now.
